generator/subaru/subaru_global_2017.dbc

```diff
@@ -53,10 +53,19 @@ BO_ 576 CruiseControl: 8 XXX
  SG_ Cruise_Activated : 41|1@1+ (1,0) [0|1] "" XXX
  SG_ Signal2 : 42|22@1+ (1,0) [0|4194303] "" XXX
 
+BO_ 803 INFOTAINMENT_STATUS: 8 XXX
+ SG_ CHECKSUM : 0|8@1+ (1,0) [0|255] "" XXX
+ SG_ COUNTER : 8|4@1+ (1,0) [0|15] "" XXX
+ SG_ SIGNAL_1 : 15|4@0+ (1,0) [0|15] "" XXX
+ SG_ SIGNAL_2 : 19|44@0+ (1,0) [0|3.51844e+13] "" XXX
```

Comment:
are these signals meaningful? if not, let's garbage collect them

Reply:
done